src/features/layer_features_panel/components/LayerFeaturesPanel/index.tsx

```diff
@@ -31,9 +31,9 @@ import type { FeatureCardCfg } from '../CardElements';
 import type { Bbox } from '~core/shared_state/currentMapPosition';
 
 export function LayerFeaturesPanel() {
-  const [currentFeatureId, { set: setCurrentFeatureId }] = useAtom(currentFeatureIdAtom);
+  const [currentFeatureId] = useAtom(currentFeatureIdAtom);
   const onCurrentChange = (id: number, feature: FeatureCardCfg) => {
-    setCurrentFeatureId(id);
+    currentFeatureIdAtom(store.v3ctx, id);
```

Comment:
no need to do it that way inside React component. use setter from atom hook https://www.reatom.dev/package/npm-react/#useatom

Reply:
Thanks, forgot about the setter function! Fixed